downloads.html

```diff
@@ -145,6 +110,41 @@ <h3 class="download-version">3.6.2<a href="#3.6.2"><i class="fas fa-link " style
         
         <h2>Archived releases</h2>
 
+            <span id="3.8.0"></span>
+            <h3 class="download-version">3.8.0<a href="#3.8.0"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+            <ul>
+                <li>
+                    Released July 29, 2024
+                </li>
+                <li>
+                    <a href="https://archive.apache.org/dist/kafka/3.8.0/RELEASE_NOTES.html">Release Notes</a>
+                </li>
+                <li>
+                    Docker image: <a href="https://hub.docker.com/layers/apache/kafka/3.8.0/images/sha256-c9aea96a4813e77e703541b1d8f7d58c9ee05b77353da33684db55c840548791">apache/kafka:3.8.0</a>.
+                </li>
+                <li>
+                    Docker Native image: <a href="https://hub.docker.com/layers/apache/kafka-native/3.8.0/images/sha256-e1b3af1f501bb1d0c2dc11ce4fb04d0132568c9da18232bdd25643b587599ded">apache/kafka-native:3.8.0</a>.
+                </li>
+                <li>
+                    Source download: <a href="https://archive.apache.org/dist/kafka/3.8.0/kafka-3.8.0-src.tgz">kafka-3.8.0-src.tgz</a> (<a href="https://archive.apache.org/dist/kafka/3.8.0/kafka-3.8.0-src.tgz.asc">asc</a>, <a href="https://archive.apache.org/dist/kafka/3.7.0/kafka-3.8.0-src.tgz.sha512">sha512</a>)
```

Comment:
Typo in https://archive.apache.org/dist/kafka/3.7.0/kafka-3.8.0-src.tgz.sha512, it should be `3.8.0` instead of `3.7.0`